### PR TITLE
Added extra_script and prerun

### PR DESCRIPTION
**Pull request by osaetin-tails**

### CODE REVIEW

**Comment by osaetin-tails** [comment]
@hkmshb the other dataportal setups had these two files but BISKIT didn't. Could this be the issue?

File: ckan/prerun.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        conn_info = connection.read()
+        conn_info = re.sub(r'"zkConnected":true', '"zkConnected":True', conn_info)
+        eval(conn_info)
+
+def init_db():
+
+    print '[prerun] Start init_db...'
+
+    db_command = ['paster', '--plugin=ckan', 'db', 'init', '-c', ckan_ini]
+    validation_command = ['paster', '--plugin=ckanext-validation', 'validation', 'init-db', '-c', ckan_ini]
```
Comment (hkmshb) [inline ckan/prerun.py]: @osaetinevbuoma Biskit doesn't use the `ckanext-validation` extension, so there is no need for `prerun.py`.

Comment (osaetin-tails) [reply ckan/prerun.py]: I'm closing this PR.

**Review by hkmshb — APPROVED** [review]
👍

**Pull request merged**